services/editor/server/api/context.js

```diff
@@ -15,16 +17,28 @@ export async function updateContext(req, res, { tweekApiHostname }, { params })
     params.identityId,
   )}`;
 
-  const response = await tweekApiClient.get(contextUrl);
-  const currentContext = response.data;
-  const newContext = req.body;
+  const patch = req.body;
+  const keysToModify = patch.map(({ op, path }) => ({
+    op,
+    key: path.replace(/^\//, '').replace(/~1/g, '/'),
```

Comment:
https://github.com/Starcounter-Jack/JSON-Patch
the operation object looks like this:
```
var documentA = {user: {firstName: "Albert", lastName: "Einstein"}};
var documentB = {user: {firstName: "Albert", lastName: "Collins"}};
var diff = jsonpatch.compare(documentA, documentB));
//diff == [{op: "replace", path: "/user/lastName", value: "Collins"}]
```
because the path can contain the `/` char, if a key in the object contains it, it's replaced with `~1`, so i'm converting it back to `/`